lib/pry/method.rb

```diff
@@ -366,6 +366,7 @@ def signature
                  when :opt   then "#{nam}=?"
                  when :rest  then "*#{nam}"
                  when :block then "&#{nam}"
+                 when :key   then "#{nam}={}"
```

Comment:
Hmm. I'm not sure about this syntax. Shouldn't it be: `#{nam}: ?` I think this syntax more closely matches what a rubyist would expect.

Also, wtf do we have the illiterate shortcuts `nam` and `typ`, ha, where did that come from? 😝 If you could also fix this to `type` and `name` that would be great

Reply:
We should also support the `:keyreq` parameter type, which are required keyword arguments. They should just be displayed as `#{name}:`

Reply:
Yeah, that all makes sense. I hammered this out in a few minutes over lunch, so I think I used `"#{nam}={}"` because it fit my exact case :) . However, I agree `"#{nam}:?"` is better. I was also wondering about the `nam`... The `typ` thing might be to prevent clashing with `type` which I think ActiveRecord uses... if memory serves correctly.